week_1/task_4_second_entry/task_4.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+#include <string>
+#include <algorithm>
+
+int main()
+{
+    std::string string_;
+    std::cin >> string_;
+    int count_ = count(string_.begin(), string_.end(), 'f');
+    if (count_ == 1) {
+        std::cout << -1;
+    }
+    else if (count_ == 0) {
+        std::cout << -2;
+    }
+    else {
+        std::cout << (string_.find('f', string_.find('f') + 1));
```

Comment:
@Ypostolov  try to use only `find ` method for this task

Reply:
Fixed in 14541a0056ce85b5708ffbe8367445899047bc35.
Please check.